src/dense.jl

```diff
@@ -1,193 +1,207 @@
 # A higher level stepper, defined as a wrapper around another stepper.
 
-#TODO: how about having an DenseStepper <: AbstractWrapper <: AbstractStepper?
-immutable DenseStepper <: AbstractStepper
-    solver::Solver
+"""
+
+Dense output options:
+
+- tspan    ::Vector{T}  output times
+- points   ::Symbol which points are returned: `:specified` only the
+  ones in tspan or `:all` which includes also the step-points of the solver.
+- stopevent  Stop integration at a zero of this function
+- roottol    TODO
+
+"""
+
+immutable DenseOptions{T<:Number} <: Options{T}
+    tspan    ::Vector{T}
+    tstop    ::T
+    # points   ::Symbol
```

Comment:
But I can't find it so I will write here: if you wrap a solver with a dense output you aren't going to specify `points=:all` anyhow.  So why bother with this option here?  On the other hand, for `odeXX` you can specify `points=:specified` or `points=:all` to wrap a stepper in a dense output method or not.